### DISCUSSION TITLE
How to make your own rotation?

### DISCUSSION THREAD

**Discussion by gandalfthecool** [Hekili/hekili#782] [Q&A]
Hey guys, tried to mess around with this but it seems i couldn't quite figure it out, is there in any straight forward explanation on how to make the rotation skills you want?

**Comment by Hekili** [comment]
No, there isn't, for some good and bad reasons.

SimulationCraft is the ecosystem for one to really test different priorities (or action priority lists / APLs).  If you test and produce an APL in SimulationCraft that outperforms the default, then importing to the addon and making any fine adjustments needed would be your next steps.

It's really challenging to refine priorities, because there are all sorts of traps that people don't think about, like opportunity costs, losing ability casts due to delaying an ability too long, etc.
